src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        ColorSupplier randColor = new ColorSupplier();
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            int random = new Random().nextInt(100);
+            System.out.println(lottery.getRandomBall(randColor.getRandomColor(),random));
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```
those values should be randomized in lottery inside, not passed here